index.js

```diff
@@ -69,6 +68,7 @@ app.use('/api/auth', authRouter);
 app.use('/api/user', userRouter);
 app.use('/api/post', verifyRole, postRouter);
 app.use('/api/comment', verifyRole, commentRouter);
+app.use('/api/admin', AdminRouter);
```

Comment:
Should have the verifyRole middleware passed in to make sure that only admin(role3) can have access to these endpoints.